### PR TITLE
Feature/admin guide provisioning corrections

### PR DESCRIPTION
**Pull request by tlaurion**
# Description

1. Android: more dns are required.
1. Android/IOS are the same but for 1. Services need to be restarted and some conf reloaded.
1. Clarifications

# Impacts
none

# Delete branch after merge
YES

### CODE REVIEW

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 1]
```diff
@@ -1546,20 +1546,29 @@ Apple devices such as iPhones, iPads, iPods and Mac OS X (10.7+) support wireles
 
 Configure the feature
 ^^^^^^^^^^^^^^^^^^^^^
+NOTE: If EAP-TLS provisioning is desired, you have to configure a PKI before going any further. Two guides exists to assist you: 'PacketFence_PKI_Quick_Install_Guide', which covers PacketFence's implementation, or 'PacketFence_MSPKI_Quick_Install_Guide' which covers Microsoft's.
```
Comment (cgx) [inline docs/PacketFence_Administration_Guide.asciidoc]: Don't use the underscores; simply put the names of the guides.

`'PacketFence PKI Quick Install Guide`' ..

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 2]
```diff
@@ -1546,20 +1546,29 @@ Apple devices such as iPhones, iPads, iPods and Mac OS X (10.7+) support wireles
...
 
-First of all, you need to configure the SSID that your devices will use after they go though the authentication process.
+First of all, you need to configure the SSID that your devices will use after they go through the authentication process.
 
-In order to do that, in the administration interface, go in 'Configuration -> Advanced Access Configuration -> Provisioners'. Then select the 'android' provisioner. Enter the SSID and save.
-
-Now do the same thing for the iOS provisioner.
+In order to do that, in the administration interface, go in 'Portal Configuration -> Provisioners'. Then select the 'android' / 'iOS' provisioner. Enter SSID relative information and roles to which the provisioner applies and save. Note that the default RADIUS certificate path is `/usr/local/pf/raddb/certs/server.crt`.
```
Comment (julsemaan) [inline docs/PacketFence_Administration_Guide.asciidoc]: ios is all lower case for the iOS provisioner

Comment (julsemaan) [reply docs/PacketFence_Administration_Guide.asciidoc]: Enter the SSID information and roles for which the provisioner applies.

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 3]
```diff
@@ -1546,20 +1546,29 @@ Apple devices such as iPhones, iPads, iPods and Mac OS X (10.7+) support wireles
...
 
   [fencing]
   passthrough=enabled
-  passthroughs=*.ggpht.com,*.googleusercontent.com,android.clients.google.com,*.googleapis.com,*.android.clients.google.com,*.gvt1.com
+  passthroughs=*.ggpht.com,*.googleusercontent.com,android.clients.google.com,*.googleapis.com,*.android.clients.google.com,*.gvt1.com,*.l.google.com,play.google.com,*.gstatic.com
+
+Then run the following commands so that passthroughs become effective:  
+
+  /usr/local/pf/bin/pfcmd service iptables restart
+  /usr/local/pf/bin/pfcmd configreload hard
```
Comment (julsemaan) [inline docs/PacketFence_Administration_Guide.asciidoc]: configreload before services restart ?

File: docs/PacketFence_Administration_Guide.asciidoc (hunk 2 above)
Comment (whitx) [inline docs/PacketFence_Administration_Guide.asciidoc]: Advanced Access Configuration

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 4]
```diff
@@ -1546,20 +1546,29 @@ Apple devices such as iPhones, iPads, iPods and Mac OS X (10.7+) support wireles
...
-Now do the same thing for the iOS provisioner.
+In order to do that, in the administration interface, go in 'Portal Configuration -> Provisioners'. Then select the 'android' / 'iOS' provisioner. Enter SSID relative information and roles to which the provisioner applies and save. Note that the default RADIUS certificate path is `/usr/local/pf/raddb/certs/server.crt`.
 
 After, you simply need to add the 'Android' and 'iOS' provisioners to your 'Connection Profile' configuration.
 
-For Android, you must allow passthroughs in your `pf.conf` configuration file:
+Android specificities
++++++++++++++++++++++
+
+For Android, you must allow passthroughs in your `pf.conf` configuration file. You might need to adapt the following to your geolocality:
```
Comment (whitx) [inline docs/PacketFence_Administration_Guide.asciidoc]: Do from the admin interface

File: docs/PacketFence_Administration_Guide.asciidoc (hunk 3 above)
Comment (whitx) [inline docs/PacketFence_Administration_Guide.asciidoc]: Configreload before iptables restart

**Comment by whitx** [comment]
Add Windows in the same section for provisioning, ios specifications (profile signing)

**Comment by whitx** [comment]
Looks good to me

**Pull request merged**